src/index.html

```diff
@@ -9,6 +9,32 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="nav__wraper">
+        <div class="nav__icon">
+          <a href="#Home">
+            <img src="images/logo.png" alt="Moyo icon">
+          </a>
+        </div>
+        <nav class="navigation">
+          <ul class="nav__list">
+            <li class="nav__item"><a href="#apple" class="nav__link is-active">Apple</a></li>
```

Comment:
Fix this with code style
```suggestion
            <li class="nav__item">
              <a href="#apple" class="nav__link is-active">Apple</a>
            </li>
```
<img width="1037" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/64750958/20ec9c63-269e-4522-911d-a043a2e7928a">